homeassistant/components/fan/zha.py

```diff
@@ -54,6 +54,11 @@ class ZhaFan(zha.Entity, FanEntity):
 
     _domain = DOMAIN
 
+    def __init__(self, **kwargs):
+        """Initiliaze ZHA fan."""
+        super().__init__(**kwargs)
+        self._only_cache = True
```

Comment:
I think you can move that to `zha.Entity`.
maybe you can make it as `self._connected` and set that after you finish to initialize that to `true`.

Reply:
Yep, will move it to `zha.Entity`.
Any particular reason for it to be `self._connected`? My idea behind this PR was to eliminate Zigbee network traffic caused by `update_before_add=True` . Since `async_added_to_hass()` is called after the very 1st `async_update()`, do you see any problems with setting the flag allowing network reads from `async_added_to_hass()`? May be something like `self._initialized`?

Reply:
Yeah, `self._initialized` is also fine